osgp/protocol-adapter-mqtt/osgp-protocol-adapter-mqtt/src/main/java/org/opensmartgridplatform/adapter/protocol/mqtt/domain/valueobjects/MqttClientDefaults.java

```diff
@@ -10,49 +10,110 @@
 package org.opensmartgridplatform.adapter.protocol.mqtt.domain.valueobjects;
 
 public class MqttClientDefaults {
```

Comment:
Merge with `MqttConfig`?

Reply:
Hmm, I don't like using a class from the `config` package within the `domain` package. Neither do I like to use `@Values` annotations in a domain class.